### ISSUE TITLE
Processing speed

### ISSUE THREAD

**Issue by zaynah91124** [HaoyuCui/WSI_Normalizer#3]
Good work! It effectively aids in image preprocessing. However, we noticed that the image processing speed significantly decreases when performing color normalization on all patches of the WSI. Could you suggest a parallel processing method or a more efficient approach for WSI color normalization?

**Comment by HaoyuCui** [comment]
Of course! You can use the deep-learning-based methods like this one: [HistAuGAN](https://github.com/sophiajw/HistAuGAN)
There's also a good paper about stain normalization [link](https://doi.org/10.1016/j.inffus.2023.101997).
The newly proposed ones support GPUs, so they can be faster and may perform better. And, of course, they are more complex.

**Comment by zaynah91124** [comment]
> Of course! You can use the deep-learning-based methods like this one: [HistAuGAN](https://github.com/sophiajw/HistAuGAN) There's also a good paper about stain normalization [link](https://doi.org/10.1016/j.inffus.2023.101997). The newly proposed ones support GPUs, so they can be faster and may perform better. And, of course, they are more complex.

Thank you for your response. I am currently focusing on the Camelyon16 and Camelyon17 multi-center datasets to investigate the impact of staining on performance. Have you worked with these datasets before? Additionally, do you have access to the Camelyon16 labels categorized by different centers? I look forward to hearing from you.

**Comment by HaoyuCui** [comment]
I'm afraid not, my current work focuses on non-public datasets. I wish you all the best with your current research.

**Comment by HaoyuCui** [comment]
> > Of course! You can use the deep-learning-based methods like this one: [HistAuGAN](https://github.com/sophiajw/HistAuGAN) There's also a good paper about stain normalization [link](https://doi.org/10.1016/j.inffus.2023.101997). The newly proposed ones support GPUs, so they can be faster and may perform better. And, of course, they are more complex.
> 
> Thank you for your response. I am currently focusing on the Camelyon16 and Camelyon17 multi-center datasets to investigate the impact of staining on performance. Have you worked with these datasets before? Additionally, do you have access to the Camelyon16 labels categorized by different centers? I look forward to hearing from you.

I'm afraid not, my current work focuses on non-public datasets. I wish you all the best with your current research.
